package.json

```diff
@@ -95,6 +95,7 @@
     "core-js": "^3.8.3",
     "cross-env": "^5.1.6",
     "del": "^5.1.0",
+    "elliptic": "^6.5.4",
```

Comment:
Why do we need to add this dependency? Can't we just re-create yarn.lock?

Reply:
>Why do we need to add this dependency
It's a  dev-dependency, so not impacting the project in any way.

>Can't we just re-create yarn.lock?
We can, but since this package is resolved from other packages and not directly from grommet's dependencies, it's bound to be erased each time we are rebuilding yarn.lock unless we specify it as a dependency.